src/test/java/org/assertj/core/presentation/StandardRepresentation_toStringOf_Test.java

```diff
@@ -12,8 +12,6 @@
  */
 package org.assertj.core.presentation;
 
-import static java.lang.String.format;
```

Comment:
I don't focus too much on imports reorganization, I let the IDE handle them, the only pain is when comparing commits but I can live with that.